### ISSUE TITLE
[FEATURE]:Update container the base image

### ISSUE THREAD

**Issue by clcc2019** [alibaba/ilogtail#1805]
Concisely describe the proposed feature：
I would like to update the base image of the ilogtail container to replace CentOS, as CentOS is no longer receiving updates and CVE vulnerabilities are not being addressed in a timely manner.

Describe the solution you'd like (if any)：
To mitigate security risks from outdated CentOS, I propose switching to a more actively maintained base image, such as ubuntu,debian,alpine,minideb, or another secure and regularly updated Linux distribution. This will ensure ilogtail stays secure and compliant with the latest security patches.

Additional comments：
By updating the base image, we can address security vulnerabilities more efficiently. This change will ensure that the ilogtail container remains robust and secure, without exposing users to known CVEs.

**Comment by yyuuttaaoo** [comment]
Yes, CentOS 7 has reached its end of life, and we are in the process of replacing it. However, this transition is introducing compatibility issues due to the newer version of glibc. Additionally, the new image must include the GCC 9 toolchain, as our projects depend on it.

**Comment by Takuka0311** [comment]
百炼自动化审查：建议保持开启。

本 Issue 请求替换已停更的 CentOS 7 容器基础镜像以修复 CVE。维护者已确认需求并指出 glibc 兼容性与 GCC 9 工具链依赖为迁移阻塞点。当前上下文与时间线未显示该变更已合入 main 或由其他 PR 解决，属于有效的核心构建与安全维护任务，需保持开启以跟踪迁移进展。.

最佳落地路径：

继续推进基础镜像迁移工作。建议采用多阶段构建策略：构建阶段使用包含 GCC 9+ 的镜像（如 Ubuntu 20.04/22.04 或 Alibaba Cloud Linux 3）以解决编译依赖；运行阶段使用轻量级镜像（如 Distroless 或 Alpine）并通过静态链接或 glibc 兼容层确保二进制文件在目标环境正常运行。待新 Dockerfile 合入 main、CI 流水线验证通过并随正式版本发布后，再关闭本 Issue。.

已核对内容：

- **maintainer_acknowledgement:** 维护者 yyuuttaaoo 于 2024-10-15 确认正在替换 CentOS 7，并明确指出受限于 glibc 版本兼容性与 GCC 9 工具链依赖，迁移工作存在技术阻塞。.
- **no_linked_pr_or_fix:** 上下文与时间线中未见关联 PR 或已合入的 Dockerfile 变更提交，核心迁移工作仍在进行中，缺乏可审计的已实现证据。.

百炼审查备注：模型 qwen3.6-max-preview；对照提交 [0fe0bf63903d](https://github.com/alibaba/loongcollector/commit/0fe0bf63903d8feee18aef282809e90c8500190e)。
